modules/users/client/controllers/settings/change-password.client.controller.js

```diff
@@ -33,11 +33,13 @@
       // If successful show success message and clear form
       $scope.$broadcast('show-errors-reset', 'vm.passwordForm');
       vm.success = true;
```

Comment:
@mleanos the couple of lines you mentioned above have been removed with no issues with tests. But there are a few tests on other controllers that check for the status of vm.success and vm.error.

```javascript
it('should set success to response message', function() {
  expect(scope.vm.success).toBe(successMessage);
});
```

Do you want me to remove all of those tests? If not can you please let me know how I can check for a DOM element from the test script?